test/e2e/cypress/e2e/checks_catalog.cy.js

```diff
@@ -27,14 +50,16 @@ context('Checks catalog', () => {
   });
 
   describe('Checks grouping and identification is correct', () => {
+    let expandedChecks = 0;
     Object.entries(groupBy(catalog, 'group')).forEach(
       ([group, checks], index) => {
         it(`should include group '${group}'`, () => {
           cy.get('.check-group > div > div > h3').should('contain', group);
         });
         it(`should expand the group '${group}' when clicked`, () => {
           index !== 0 && cy.get('.check-group').contains(group).click();
-          cy.get('div.check-row').should('have.length', (index + 1) * 2);
+          expandedChecks = checksInGroup[group] + expandedChecks;
+          cy.get('div.check-row').should('have.length', expandedChecks);
```

Comment:
I don't have a clear mind about what this is but having a `let` and reassigning it inside an `it` block sounds a little but funky

Reply:
I don't like that either :smile: 
Changed here d303fb8122084724be35bc9968ff04d8c701e951

Reply:
Aw yeah